client/starwhale/core/instance/view.py

```diff
@@ -44,6 +48,68 @@ def login(self, instance: str, alias: str, **kw: str) -> None:
             )
             sys.exit(1)
 
+    def login_with_browser(self, instance: str, alias: str) -> None:
+        p = urlparse(instance)
+        instance = f"{p.scheme}://{p.netloc}"
+
+        import uvicorn
+        from fastapi import FastAPI
+        from starlette.middleware.cors import CORSMiddleware
+
+        login_done = False
+
+        app = FastAPI()
+        app.add_middleware(
+            CORSMiddleware,
+            allow_origins=["*"],
+            allow_credentials=True,
+            allow_methods=["*"],
+            allow_headers=["*"],
+        )
+
+        @app.on_event("startup")
+        async def on_startup() -> None:
+            nonlocal instance
+            url = f"{instance}/auth/client"
+            import webbrowser
+
+            webbrowser.open(url)
+
+        @app.get("/login")
+        async def _login(token: str) -> dict:
+            print(f"get token {token}")
```

Comment:
How about use console to logging

Reply:
`console.info` did not work, I don't know why :rofl: 
I'll try to find out the reason

Reply:
I forgot to set the log lv using `-vvv`  :face_with_spiral_eyes: 
removed this line, and change the success echo.